framework/set/provisioning/setProvidersAndUsersTF.go

```diff
@@ -42,5 +44,27 @@ func SetProvidersTF(rancherConfig *rancher.Config, terraformConfig *config.Terra
 	provBlockBody.SetAttributeValue("token_key", cty.StringVal(rancherConfig.AdminToken))
 	provBlockBody.SetAttributeValue("insecure", cty.BoolVal(*rancherConfig.Insecure))
 
+	rootBody.AppendNewline()
```

Comment:
nit; the file name changed and behavior of this function changed, but not the function name;

Also this function is getting a little long and expanded scope. I think it would be worth splitting the user section out into its own helper for clarity.

Reply:
I can update the function name, that was an oversight. To the second point, I think that's outside the scope. Josh and I have talked of that in general, not just in this function.

Several files are getting pretty long, I think it would be worth it to have a separate PR where we further break down functions for readability.